Source/WebCore/rendering/RenderFlexibleBox.h

```diff
@@ -265,11 +265,11 @@ class RenderFlexibleBox : public RenderBlock {
 
     // This is used to cache the preferred size for orthogonal flow children so we
     // don't have to relayout to get it
-    HashMap<SingleThreadWeakRef<const RenderBox>, LayoutUnit> m_intrinsicSizeAlongMainAxis;
+    SingleThreadWeakHashMap<const RenderBox, LayoutUnit> m_intrinsicSizeAlongMainAxis;
     
     // This is used to cache the intrinsic size on the cross axis to avoid
     // relayouts when stretching.
-    HashMap<SingleThreadWeakRef<const RenderBox>, LayoutUnit> m_intrinsicContentLogicalHeights;
+    SingleThreadWeakHashMap<const RenderBox, LayoutUnit> m_intrinsicContentLogicalHeights;
```

Comment:
I have a better fix in the commit queue at https://github.com/WebKit/WebKit/pull/22934.